packages/client/src/pages/Home/index.js

```diff
@@ -98,6 +112,24 @@ const HomepageBody = styled.div`
   }
 `;
 
+const howItWorks = [
+  {
+    key: 1,
+    step: 'home.howItWorks.step1',
+    imgsrc: step1
+  },
+  {
+    key: 2,
+    step: 'home.howItWorks.step2',
+    imgsrc: step2
+  },
+  {
+    key: 3,
+    step: 'home.howItWorks.step3',
+    imgsrc: step3
+  }
+]
+
 const Subjects = [
```

Comment:
I feel the key field is not needed, you can just do something like map((subject, index) => <Component key={text + key}/>